subjects/devops/serverless/README.md

```diff
@@ -0,0 +1,159 @@
+
+## Level 0: Basic Lambda Function hosted via Serverless
+
+**1. Create Slack Bot**
+
+Write a slack bot, that sends a reminder to slack channel with the following message:
+
+```sh
+Dear Board Members! This is a reminder to make the payment for the licenses of the software. The due date is 07.XX.YY, where XX is the month and YY is the year. The amount to be paid is $ZZZ.VV. Please make the payment as soon as possible to <IBAN_NUMBER>. Thank you!
+```
+
+The environment variables which must be available in the Lambda function are:
+
+- `SLACK_WEBHOOK_URL` - The Slack Webhook URL
+- `AMOUNT` - The amount to be paid
+- `IBAN_NUMBER` - The IBAN number
+
+The due date must be calculated based on the current date for the next month.
+
+Before step two, you need to test that Slack Bot is functioning properly by running an application locally. It should send a message to the Slack channel.
+
+**2. Write tests for the application**
+
+Write integration tests for the application to ensure that each part of the application is functioning properly.
+
+**3. Use [Serverless Framework](https://github.com/serverless/serverless) to host the lambda function**
+
+Serverless framework is an automation tool used to deploy serverless applications which can help with event-driven architecture deployments.
+
+Use the serverless framework to host the lambda function combined with AWS Eventbridge.
+
+**Refs**
+
+- [AWS EventBridge](https://www.serverless.com/framework/docs/providers/aws/events/event-bridge)
+- [AWS Lambda Function](https://www.serverless.com/framework/docs/providers/aws/guide/functions).
+
+AWS Eventbridge should trigger lambda function every month on the 1st day of the month at 10:00 AM.
+
+**Note** Before hosting, you need to ensure that the bot is written in a way that it can be hosted on AWS Lambda. You can look at the example [here](https://github.com/KostLinux/aws-incident-manager-notifier/blob/56d52e90f8a14e689e7d2a1c7ee44590de5af2f5/main.go#L158).
+
+## Level 1: Basic Lambda Function automation via Cloudformation
+
+**1. Repeat the 1st and 2nd step from Level 0**
+
+Create a slack bot, that sends a reminder to slack channel. Write integration tests for the application to ensure that each part of the application is functioning properly.
+
+**Note** Don't forget to add Lambda handler for the application
+
+**2. Use [Cloudformation](https://aws.amazon.com/cloudformation/) to host the lambda function**
+
+Cloudformation is Infrastructure As Code (IaC) tool used to automate the provisioning of AWS resources inside the AWS. It allows you to use a simple yaml syntax to define the resources you want to create.
+
+**2.1 Hosting the Lambda function**
+
+Write cloudformation template with parameters (variables) `SlackWebhookUrl`, `Amount`, `IbanNumber`. The template should automatically provision the Slackbot based on architecture mentioned in [Level 0](#level-0-basic-lambda-function-hosted-via-serverless) step 3.
```

Comment:
The link to the section is not correct.
#level-0-basic-lambda-function-hosted-via-serverless : 404